public/css/theme/main.scss

```diff
@@ -10,8 +10,12 @@
   &.open {
     .UNX-facet__list {
       max-height: 500px;
+      overflow: auto;
+    }
+    .UNX-facet__list.UNX-facet__listShowLimited{
+      max-height: 90px !important;
```

Comment:
Can we avoid the important property here. This would give the integrator more flexibility to style according to his needs.

Reply:
ok